packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -146,6 +147,50 @@ describeCompat("Flushing ops", "NoCompat", (getTestObjectProvider, apis) => {
 		await provider.ensureSynchronized();
 	}
 
+	it("can send and a batch containing a future/unknown op type", async () => {
+		await setupContainers({
+			flushMode: FlushMode.TurnBased,
+			compressionOptions: {
+				minimumBatchSizeInBytes: 10,
+				compressionAlgorithm: CompressionAlgorithms.lz4,
+			},
+			enableGroupedBatching: true,
+			chunkSizeInBytes: 100,
+		});
+		const dataObject1BatchMessages: ISequencedDocumentMessage[] = [];
+		const dataObject2BatchMessages: ISequencedDocumentMessage[] = [];
+		setupBatchMessageListener(dataObject1, dataObject1BatchMessages);
+		setupBatchMessageListener(dataObject2, dataObject2BatchMessages);
+
+		// Submit two ops, one of which is unrecognized
+		dataObject2map1.set("key1", "value1");
+		(
```

Comment:
This is so hard to read. I thought this was doing a set().submit() and got so confused.